pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cookiecutter-nautobot-app"
-version = "1.0.0"
+version = "2.3.0"
```

Comment:
Was there a reason the version number wasn't being bumped in the pyproject.toml?

Reply:
Let's change the mode: https://python-poetry.org/docs/basic-usage/#operating-modes